JASP-Engine/JASP/R/exploratoryfactoranalysis.R

```diff
@@ -17,8 +17,9 @@
 
 
 .DotIfNULL <- function(x){
-  
-  if (is.null(x) || any(is.na(x)) || !is.finite(x)){
+
```

Comment:
@hummyhummy Already defined in pca. What is x exactly?

Reply:
removed and comments added